vision/spec/detect_safe_search_spec.rb

```diff
@@ -0,0 +1,53 @@
+# Copyright 2016 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in write, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "rspec"
+require "google/cloud/storage"
+
+require_relative "../detect_safe_search"
+
+describe "Detect Safe Search Properties" do
+
+  before do
+    @storage    = Google::Cloud::Storage.new
```

Comment:
This should use project: ENV["GOOGLE_CLOUD_PROJECT"].. if you're going to use @project_id = ENV["GOOGLE_CLOUD_PROJECT"].